apps/src/music/player/sound.js

```diff
@@ -14,6 +16,7 @@ var audioSystem = null;
 export function InitSound(desiredSounds) {
   // regular web version.
   baseSoundUrl = 'https://curriculum.code.org/media/musiclab/';
+  restrictedSoundUrl = '/restricted/musiclab/';
```

Comment:
nit:
This is a URL path.

since you have two variables called URL next to each other and they have different format, I think the distinction is important

Reply:
Good call! I'll update